test/integration/connect/envoy/docker-compose.yml

```diff
@@ -175,7 +175,7 @@ services:
   s1-sidecar-proxy:
     depends_on:
       - consul-primary
-    image: "envoyproxy/envoy:v${ENVOY_VERSION:-1.8.0}"
+    image: "envoyproxy/envoy:v${ENVOY_VERSION:-1.13.0}"
```

Comment:
Good catch on these. Overall I'm not sure if we should just remove this default since it's annoying to have to update and we always set an ENVOY_VERSION in our test runs anyway...

If there's a downside to removing this, can we add to the checklist to come and update all these? Or is there a way in docker-compose files to set a variable or something?

Reply:
💯 lets just take it out.